docs/administering/faq.md

```diff
@@ -516,60 +516,10 @@ that we provide. Pay special attention to the `RewriteRule` stanzas.
-in a minute of usage. Generating 20 hostnames would trigger the [rate
-limits](https://letsencrypt.org/docs/rate-limits/), at which point all other visitors to Sandstorm
-would be unable to use grains. This issue alone makes the problem intractable.
-
-There are some additional problems as well.
-
-- **Loss of defense-in-depth for cross-site-request forgery protection and more.** Let's Encrypt
-  shares all certificate hostnames to the Certificate Transparency logs, which means you lose one of
-  Sandstorm's defense-in-depth mechanisms. Read the doc on [why we use wildcards](wildcard.md) for
-  more information on defense-in-depth via wildcards.
-
-- **Loss of privacy for static publishing URLs.** When users generate [static publishing
-  content](../developing/web-publishing.md) within a grain, they typically expect it to be private
-  until they choose a well-known domain name for the content. Let's Encrypt's integration with
-  Certificate Transparency would make all static publishing URLs by default.
-
-Some subproblems seem to have solutions, but they do not add up to a full solution for now.
-
-- **Latency.** The most straightforward way to implement Let's Encrypt dynamic provisioning with
-  Sandstorm would be for each new domain to be provisioned at the time it is created within
-  Sandstorm. This would mean that a visitor to your Sandstorm server would perceive a >10 second
-  latency introduced by Let's Encrypt while Let's Encrypt validates the domain and provides a
-  certificate. This could be solved by pre-generating a large number of subdomain certificates, but
-  it's not feasible to pregenerate enough. Consider that a server with 20 users and 20 grains would
-  need at least 400 certificates, but Let's Encrypt permits 20 certificates per week at the time of
-  writing. Additionally, Sandstorm periodically cycles these hostnames out of use. Let's Encrypt's
-  support for SAN certificates would allow 2,000 hostnames per week, but if a grain suddenly became
-  popular, new visitors would overwhelm this rate limit.
-
-One approach is to use `*.example.sandcats.io` for wildcard hostnames, but use a `BASE_URL` of `example.com`
-where `example.com` is certified via Let's Encrypt. However, this would mean that `example.com` contains
-IFRAME elements pointing at `*.example.sandcats.io` and these IFRAMEs may not be able to set cookies within
-the subdomain due to the increasingly-common **third-party cookie-blocking** done by browsers such as Chrome
-and Firefox.
+Yes, support for Let's Encrypt was added in 0.263. To use Let's Encrypt, your Sandstorm server needs
+to be able to modify your DNS records. This is made possible by [the ACME.js library](https://git.rootprojects.org/root/acme.js) as long as
+you use a supported DNS provider (Sandcats.io, Cloudflare, Digital Ocean, DNSimple, Duck DNS, GoDaddy,
+Gandi, Namecheap, Name.com, AWS Route 53, or Vultr). You can now configure this in the admin panel.
```

Comment:
A couple thoughts:

- I don't think we need to call out what library we're using here.
- We should mention that it's possible to manually upload a cert if needed (useful if the user's DNS provider is not supported), though the user will need to keep it up to date themselves.

Reply:
My thought is that the library retains relevant regarding issues with or interest in the supported providers.

I didn't want to go crazy about things like manual upload options here because this is just an FAQ and I wanted to purge the suggestion that it can't be done.

Perhaps the right thing to do would be to come back and add a link here to the HTTPS docs page when that is revised as well?

Reply:
I think given that you also explicitly list the supported providers it's still not important. But otherwise I'm not too picky; I agree the most important thing is to remove the incorrect information.

Reply:
I am on mobile now, I will merge and then revise it again alongside the HTTPS page update and remove the extraneous information.